src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/PearBomTool.groovy

```diff
@@ -74,9 +77,9 @@ class PearBomTool extends BomTool {
         ExecutableOutput pearDependencies = executableRunner.runExe(pearExePath, 'package-dependencies', PACKAGE_XML_FILENAME)
 
         File packageFile = detectFileManager.findFile(sourcePath, PACKAGE_XML_FILENAME)
-        def packageXml = new XmlSlurper().parseText(packageFile.text)
+        GPathResult packageXml = new XmlSlurper().parseText(packageFile.text)
         String rootName = packageXml.name
-        String rootVersion = packageXml.version.release
+        String rootVersion = packageXml.version.result
```

Comment:
Why did you change this from `.release` to `.result`? If you look at the package.xml file in src/test/resources for Pear, it should be `.release`